cookbooks/bcpc-hadoop/recipes/oozie.rb

```diff
@@ -155,15 +158,14 @@
   block do
     if not system " #{cmd} 'SELECT SCHEMA_NAME FROM INFORMATION_SCHEMA.SCHEMATA WHERE SCHEMA_NAME = \"oozie\"' | grep oozie" then
       code = <<-EOF
-                CREATE DATABASE oozie;
-                GRANT #{privs} ON oozie.* TO 'oozie'@'%' IDENTIFIED BY '#{get_config('mysql-oozie-password')}';
-                GRANT #{privs} ON oozie.* TO 'oozie'@'localhost' IDENTIFIED BY '#{get_config('mysql-oozie-password')}';
-                FLUSH PRIVILEGES;
+        CREATE DATABASE oozie;
+        GRANT #{privs} ON oozie.* TO 'oozie'@'%' IDENTIFIED BY '#{get_config('mysql-oozie-password')}';
+        GRANT #{privs} ON oozie.* TO 'oozie'@'localhost' IDENTIFIED BY '#{get_config('mysql-oozie-password')}';
+        FLUSH PRIVILEGES;
       EOF
       IO.popen("mysql -uroot -p#{get_config('mysql-root-password')}", "r+") do |db|
         db.write code
       end
-      #system "sudo -u oozie #{OOZIE_CLIENT_PATH}/bin/ooziedb.sh create -sqlfile #{OOZIE_CLIENT_PATH}/oozie.sql -run Validate DB Connection"
       system "sudo -u oozie /usr/hdp/current/oozie-server/bin/ooziedb.sh create -sqlfile /usr/hdp/current/oozie-server/oozie.sql -run Validate DB Connection"
```

Comment:
Can't use `#{OOZIE_CLIENT_PATH}` since the file `ooziedb.sh` moved from that path to `/usr/hdp/current/oozie-server/bin/ooziedb.sh`:
```bash
bcpc-vm2:~$ find /usr/hdp -type f -name ooziedb*
/usr/hdp/2.2.0.0-2041/oozie/bin/ooziedb.sh
bcpc-vm2:~$ ls -l /usr/hdp/current/oozie-server
lrwxrwxrwx 1 root root 27 Nov 24 15:14 /usr/hdp/current/oozie-server -> /usr/hdp/2.2.0.0-2041/oozie
```

That is why the command with `#{OOZIE_CLIENT_PATH}` was commented out. I merely deleted the commented out code in this commit.